int32 cmd
int32 speed
bool  use_double_echo
---
int32 status
